bitnami/elasticsearch/values.yaml

```diff
@@ -79,6 +79,75 @@ image:
   ## Set to true if you would like to see extra information on logs
   ##
   debug: false
+
```

Comment:
Review that ingest and coordinating values don't have references to "Deployment"

Reply:
Fixed a single reference to Deployment instead of Stateful.